src/langcheck/metrics/eval_clients/_base.py

```diff
@@ -222,3 +225,35 @@ def compute_metric_values_from_template(
             metric_values=scores,
             language=language,
         )
+
+    def augment_from_template(
```

Comment:
Even though this method can take arbitrary `template` as the argument, it actually assumes that `template` only accept the specific `instance` variable. How about

- make `instances` argument `list[dict[str, str]]` such as `[{'instance': 'Hello, world!!'}, ...]` 
- map them as `render(**instance)`

In that way, we can also update the method name to more general one like `repeat_requests` or something.

Tell me your opinion!!

Reply:
Ah this is what we talked offline before, thanks for noting this again!
That sounds good, let me name it as `repeat_requests_from_template` then!